### ISSUE TITLE
AppVeyor changes file encoding or replaces Environment.NewLine

### ISSUE THREAD

**Issue by MikhailTymchukDX** [appveyor/ci#642]
It's pretty simple to reproduce: add unit test where a string is broken into parts, like so:

![image](https://cloud.githubusercontent.com/assets/17080984/13011212/c2e4300e-d1b6-11e5-9826-28a7efbd9649.png)

This tests passes locally, but fails on AppVeyor: https://ci.appveyor.com/project/Skybladev2/appveyortest/build/tests

**Comment by FeodorFitsner** [comment]
This could be easily fixed with changing the way git handles CRLFs: https://github.com/FeodorFitsner/AppVeyorTest-1/blob/master/appveyor.yml#L1-L2

    init:
    - git config --global core.autocrlf false

**Comment by MikhailTymchukDX** [comment]
@FeodorFitsner Thank you for answer, but it helped me only in for the simplest project.
I changed Clone script for AppVeyorTest and test passed: https://ci.appveyor.com/project/Skybladev2/appveyortest/build/1.0.25/tests
But when I tried the same in real-life repository nothing changed (see last red test): https://ci.appveyor.com/project/Skybladev2/ajaxcontroltoolkitmock/build/1.0.14/tests

**Comment by FeodorFitsner** [comment]
`git config --global core.autocrlf false` means repo will be downloaded "as is" without "fixing" CR LF.

I've downloaded a raw copy of [ToolkitResourceManagerTests.cs](https://raw.githubusercontent.com/MikhailTymchukDX/AjaxControlToolkitMock/master/AjaxControlToolkit.Tests/ToolkitResourceManagerTests.cs). This is what you have there:

![image](https://cloud.githubusercontent.com/assets/5041459/13056700/c743f0e6-d3cb-11e5-8335-bef6e24ef604.png)

Check what global (or repo-level in .gitattributes) settings you have locally - most probably CRLF is changed on commits.

**Comment by MikhailTymchukDX** [comment]
Wow, did not see, that files actually have LF endings. I googled what happened and found out [this answer](http://stackoverflow.com/a/15967109/644496). 
Git actually replaces CR LF with LF on commit, but tests do not know about it and use wrong, modified versions of files when running on AppVeyor.
I changed the command to `git config --global core.autocrlf true` to quickly fix it.
Thanks for clarifying, that was very informative.
